### ISSUE TITLE
Palette 3 and Klipper

### ISSUE THREAD

**Issue by audiorazor** [tomvandeneede/p2pp#110]
Do you know if the new palette will work with p2pp?

**Comment by tomvandeneede** [comment]
Hi
tjis is a question best asked in the Mosaic groups on fb.  I don’t have insight on the controller but given it is an extension of standard gcode I don’t think it will  be an issue

**Comment by Yezariael75** [comment]
Would like to bring this up, is there any chance for Klipper? Thanks!

**Comment by tomvandeneede** [comment]
Currently only accessory mode, working on direct mode but on hold pending p3 firmware upgrade

**Comment by ThomasNyk** [comment]
Any update on this? What are we waiting on? Mosaic, klipper or a third entity?

**Comment by tomvandeneede** [comment]
Waiting for Mosaic to allow remote pings through MQTT
it is pending work with lower priority on their side  since they are workington the release of the Element and Array devices
Hopefully they will get the update out soon after the release
should not be a lot of work I would guess
